lib/pages/chat/events/message/message_style.dart

```diff
@@ -139,4 +139,6 @@ class MessageStyle {
   static const double pushpinIconSize = 14.0;
 
   static const double paddingAllPushpin = 0;
+
+  static const double contextMenuWidth = 250;
```

Comment:
IMO it is too big

Reply:
<img width="796" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/43041967/26d7bc89-2626-4c45-a20e-723b81d91e6b">
The design have even bigger width